frontend/src/static/css/style.scss

```diff
@@ -293,6 +293,13 @@ header {
     &--contribution {
       display: inline;
     }
+
+    &--button {
```

Comment:
I thought we got this button CSS at another place? Can't we reuse it?

Reply:
We cannot reuse it since it's inside another CSS. We can extract it out from there and create a global button. Want me to do that?

Reply:
I think it is the button that used a lot in v_summary, can it be extracted out to be at v_summary level?

Reply:
Yes, I just checked. Can be extracted to the v_summary level

Reply:
Making the necessary changes.

Reply:
Actually, I just realized that both have different padding requirements. Better to keep them separate

Reply:
Let me know how you want me to proceed.